### DISCUSSION TITLE
Keep the 0x for Probe always there?

### DISCUSSION THREAD

**Discussion by mengstr** [hneemann/Digital#1003] [General]
Is there a way to get the "0x"-part to always be displayed? As it is now the digits jumps all over the place when the app thinks it can remove the 0x since one or more of the digits A..F is present in the number.
 
https://user-images.githubusercontent.com/325326/181384503-0e105839-9d57-48f4-9c8f-ea0f2dda33b7.mov

**Comment by hneemann** [comment]
Why not selecting the "hex" output mode?

**Reply by mengstr** [reply]
Me so stupid. :) 
I just assumed it was in hex mode already since it displayed hex digits. But the default mode apparently is hex with 0x only when needed - which is a kinda confusing default mode - at least for me...
